README.md

```diff
@@ -13,9 +13,7 @@ It took about three hours for Racket to grow on me. I know with a fair amount of
 
 ### Installation
 
-Cloning the repository directly into your Packages directory is simplest.
-
-    git clone https://github.com/andkerosine/rainbowth.git Rainbowth
+Clone this repository directly into your Packages directory.
```

Comment:
Made this change because the original repo (which you forked) didn't work for me. Took me a bit to realize I wasn't actually installing your version, despite following "your instructions".